packages/turf-line-split/test.js

```diff
@@ -33,6 +33,16 @@ test('turf-line-split', t => {
     t.end();
 });
 
+test('turf-line-split -- lines should split the same feature 1 with 2 as 2 with 1', t => {
+    const geojson = load.sync(directories.in + 'linestrings.geojson');
+    const featureOne = geojson.features[0];
+    const featureTwo = geojson.features[1];
+
+    const resultsOne = lineSplit(featureOne, featureTwo);
+    const resultsTwo = lineSplit(featureTwo, featureOne)
+    t.deepEquals(resultsOne, resultsTwo, 'Splits were made in different locations')
+})
```

Comment:
Note: Add `t.end();` when using Tape tests.

Reply:
Oops, sorry. Will sort it.